src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -549,4 +557,48 @@ default String killCountMessage() {
         return "%USERNAME% has defeated %BOSS% with a completion count of %COUNT%";
     }
 
+    @ConfigItem(
+        keyName = "combatTaskEnabled",
+        name = "Enable Combat Tasks",
+        description = "Enable notifications for combat achievements",
+        position = 43,
+        section = combatTaskSection
+    )
+    default boolean notifyCombatTask() {
+        return false;
+    }
+
+    @ConfigItem(
+        keyName = "combatTaskSendImage",
+        name = "Send Image",
+        description = "Send image with the notification",
+        position = 44,
+        section = combatTaskSection
+    )
+    default boolean combatTaskSendImage() {
+        return false;
+    }
+
+    @ConfigItem(
+        keyName = "combatTaskMinTier",
+        name = "Min Tier",
+        description = "Minimum combat achievement tier to warrant a notification",
+        position = 45,
+        section = combatTaskSection
+    )
+    default CombatAchievementTier minCombatAchievementTier() {
+        return CombatAchievementTier.EASY;
+    }
+
+    @ConfigItem(
+        keyName = "combatTaskMessage",
+        name = "Notification Message",
+        description = "The message to be sent to the webhook. Use %USERNAME% to insert your username, %TIER% to insert the task tier, %TASK% to insert the task name",
+        position = 46,
+        section = combatTaskSection
+    )
+    default String combatTaskMessage() {
+        return "%USERNAME% has completed a combat task of %TIER% difficulty: %TASK%";
```

Comment:
```suggestion
        return "%USERNAME% has completed %TIER% combat task: %TASK%";
```
`felanbird1 has completed Master combat task: Lightning Lure`